Cargo.toml

```diff
@@ -23,6 +23,6 @@ default-features = false
 [dev-dependencies]
 compiletest_rs = "0.2.1"
 docopt = "0.7"
-futures = "0.1.7"
+futures = "0.1.14"
```

Comment:
I missed this dependency in the shuffle, but I think it can just go away, as rayon-futures now tests itself.